client.go

```diff
@@ -39,6 +35,17 @@ type GetResult struct {
 	Dst string
 }
 
+type getError struct {
+	// when fatal is true it means that something when wrong with
```

Comment:
It would be good to have this comment added to any exported method that returns a getError so that it is clear what to expect from the error. Or maybe just export this type as in `GetError` and add the comment above the type so that it is available via godoc. What do you think?

Reply:
Right now this one is only returned by the `get` method which is not exported. Do you think exporting `getError` is necessary anyways?
I'm updating the comment to make it clear that is used by `get`.

Reply:
Oh in that case you can disregard my comment. I don't think exporting it is necessary if its only used by unexported methods.